CHANGELOG.md

```diff
@@ -1,76 +1,79 @@
-## 0.3.28
+All notable changes to this project will be documented in this file.
+We follow the [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0) format.
```

Comment:
We should mention we use keepachangelog formatting and also put the version in brackets. Also add an ## Unreleased heading. There are other small adjustments in the keepachangelog guide.
Using asterisk is totally fine for listings, better than dashes

Reply:
I think using dashes would be good, as this is used in the example changelog on keepachangelog.com and using it reduces the probability of adding problems when someone decides to parse it. It's the same for markdown, so no visible change to people reading it on github and it's good enough when read as plain text

Reply:
also, since we now have the `[Unreleased]` section, is it part of this PR to add the changelogs from the merged PRs to it? I'd say it is

Reply:
added them now, will have to add them to the currently open PRs as well